### DISCUSSION TITLE
while displaying the [id] , instead of id i am getting some image.

### DISCUSSION THREAD

**Discussion by laskarDipankar** [vercel/community#7374] [Bug Report]
### Page to Investigate

#

### Steps to Reproduce

import React from "react";

interface IParams {
  id: string;
}

const page = ({ params }: { params: IParams }) => {
  const { id } = params;

  console.log(id);
  return (
    <div className="mt-20 text-2xl flex justify-center">
      page {params.id.toString()} {id}
    </div>
  );
};

export default page;

the issue occurred probably because i was using airstone font which I downloaded externally ,

![Screenshot 2024-07-05 194814](https://github.com/vercel/community/assets/114591820/e4084cad-d617-42bb-bb9c-f7e57887b87d)

is there a solution for this . how should i handle params in this scenario.

thanks.

**Comment by pawlean** [comment]
Hi @laskarDipankar,

It's difficult to help you debug this, could you share a repo with us?

**Comment by laskarDipankar** [comment]
Sure, here it is.


https://github.com/mundanecoder/music-composer

**Comment by laskarDipankar** [comment]
if you route to /collection/1 you will see the error

[a comment by an automated account is not shown]
